plover/gui/config.py

```diff
@@ -200,20 +202,30 @@ def __init__(self, config, parent):
         sizer = wx.FlexGridSizer(2, 3)
         sizer.AddGrowableCol(1)
 
-        sizer_flags = wx.SizerFlags(1).Align(wx.ALIGN_CENTER_VERTICAL).Border(wx.ALL, UI_BORDER)
+        sizer_flags = wx.SizerFlags(1) \
+            .Align(wx.ALIGN_CENTER_VERTICAL) \
+            .Border(wx.ALL, UI_BORDER)
 
-        sizer.AddF(wx.StaticText(self, label=MACHINE_LABEL), sizer_flags.Left())
+        sizer.AddF(
```

Comment:
I tend to dislike indentation-as-alignment vs indentation-as-relative-level. Former really does not play well with tabs (since someone will indent to a mix of tabs and spaces for their specific tab=4 sp, then I'll view with tab=8 sp and it's all crazy), latter works great no matter how you're indenting.